sci-biology/open-ephys-gui/open-ephys-gui-0.5.2.2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="Processing, recording, and visualizing multichannel ephys data"
+HOMEPAGE="http://www.open-ephys.org/gui/"
+LICENSE="GPL-3"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="master"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+elif [[ ${PV} == "99999999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="development"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+else
+    SRC_URI="https://github.com/open-ephys/plugin-GUI/archive/v${PV}.tar.gz"
+	Suffix=${PV}
+	SubDir="plugin-GUI-${PV}"
+	S="${WORKDIR}/$SubDir"
+fi
+
+SLOT="${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="jack +KWIKFormat +NetworkEvents +NWBFormat"
+
+BDEPEND="
+	<sys-devel/gcc-9
+"
+DEPEND="
+	media-libs/alsa-lib
+	media-libs/freeglut
+	media-libs/freetype
+	x11-libs/libXrandr
+	x11-libs/libXcursor
+	x11-libs/libXinerama
+	jack? ( media-sound/jack-audio-connection-kit )
+	KWIKFormat? ( >sci-libs/hdf5-1.8.12[cxx(+)]	)
+	NetworkEvents? ( =net-libs/zeromq-3* )
+	NWBFormat? ( <sci-libs/hdf5-1.8.21 =sci-libs/hdf5-1.8*	)
+"
+RDEPEND="${DEPEND}"
+
+BUILD_DIR="$S/Build"
+PATCHES=( "${FILESDIR}"/${P}.patch )
+
+src_prepare() {
+	cmake_src_prepare
+
+	use jack && sed -i 's/JUCE_APP_VERSION=/JUCE_JACK=1\n    JUCE_APP_VERSION=/' "${WORKDIR}/$SubDir/CMakeLists.txt" || die "Sed failed!"
+}
+
+src_install() {
+	dodir opt/open-ephys-$Suffix
+	cp -R "${BUILD_DIR}"/Gentoo/* "${ED}"/opt/open-ephys-"$Suffix"/
+	dosym /opt/open-ephys-"$Suffix"/open-ephys usr/bin/open-ephys-"$Suffix"
```

Comment:
You can use relative paths

Reply:
You mean `opt/open-ephys-"$Suffix"/open-ephys` instead of `/opt/open-ephys-"$Suffix"/open-ephys`?

Reply:
Yes, that should work. Best to also try it out, though :)

Reply:
It didn't, sorry :/ see comment below.